pkg/controller/channel/channel_controller.go

```diff
@@ -61,10 +67,13 @@ var (
 
 	//DeployableAnnotation is used to indicate a resource as a logic deployable
 	DeployableAnnotation = dplv1.SchemeGroupVersion.Group + "/deployables"
+	srtGvk               = schema.GroupVersionKind{Group: "", Kind: "Secret", Version: "v1"}
+	cmGvk                = schema.GroupVersionKind{Group: "", Kind: "ConfigMap", Version: "v1"}
```

Comment:
The above two can be consts?

Reply:
`schema.GroupVersionKind` this guy is variable, so I can't put them into const